dss/api/search.py

```diff
@@ -63,14 +67,24 @@ def post(json_request_body: dict, replica: str, per_page: int, _scroll_id: typin
 
         # TODO: (tsmith12) allow users to retrieve previous search results
         _scroll_id = page['_scroll_id']
-        result_list = [{
-            'bundle_id': hit['_id'],
-            'bundle_url': _build_bundle_url(hit, replica),
-            'search_score': hit['_score']
-        } for hit in page['hits']['hits']]
+
+        if format == 'raw':
```

Comment:
I would consider switching this to a simple for-loop because you can remove a fair amount of the duplicated code.